components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -387,6 +394,72 @@ - (CGFloat)estimatedTextViewLineHeight {
   return MDCCeil(self.textView.font.lineHeight * 2.f) / 2.f;
 }
 
+#pragma mark - Trailing View Implementation
+
+- (void)updateTrailingViewLayout {
+  if (!self.trailingView) {
+    return;
+  }
+
+  self.trailingView.translatesAutoresizingMaskIntoConstraints = NO;
+
+  if (![self.trailingView isDescendantOfView:self]) {
+    [self addSubview:self.trailingView];
+  }
+
+  if (!self.trailingViewTrailing) {
```

Comment:
Once created, will self.trailingViewTrailing ever be nil again?
dispatch_once ?
(dispatch_once may be less clear)
Perhaps a comment?
// Create trailing view trailing constraint if it doesn't exist yet.
If this is only created once, will it break if we change self.trailingView?